python/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 3.10)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
(same question) I'm just curious; is `compile_commands.json` needed to support pip installation?